index.js

```diff
@@ -1,61 +1,75 @@
 const url = 'https://api.themoviedb.org/3/movie/12?api_key=7c4a986b16b3b892bd7111a358d63e05&language=en-US';
 const logo = 'https://api.themoviedb.org/3/movie/75780?api_key=7c4a986b16b3b892bd7111a358d63e05&language=en-US'
 const path = 'https://image.tmdb.org/t/p/w1920_and_h600_multi_faces'
+const imageCardPath = 'https://image.tmdb.org/t/p/w185'
 const latestRelease = 'https://api.themoviedb.org/3/trending/all/day?api_key=7c4a986b16b3b892bd7111a358d63e05'
 const pathMovie = 'https://api.themoviedb.org/3/search/movie?api_key=7c4a986b16b3b892bd7111a358d63e05&query='
 const inputBar = document.getElementById('form');
 const collection = 'https://www.themoviedb.org/collection/';
+const highScore = 70;
+const regularScore = 50;
+const color = {
+    green: '#58FA58',
+    blue: '#F3F781',
+    red: '#FA5858'
+};
 
 function getRandomInt(max) {
     return Math.floor(Math.random() * max);
 }
 
 inputBar.addEventListener('submit', function(e) {
+
     e.preventDefault();
-    
     let name = inputBar[0].value;
     const nameCleaned = name.replace(/\s/g, '+');
     let search = pathMovie+nameCleaned;
-    console.log(search);
 
     fetch(search)
     .then(response => response.json())
     .then((data => {
-        let id = data.results[0].id;
-        let result = collection+id;
-        window.location = result;
+        renderResults(data.results);
     }))
-
     inputBar.reset();
-    
 
 })
 
- function goSearch() { 
-    
+ const renderResults = function(data) {
+
+    const resultsDiv = document.getElementById('containerResults');
+    resultsDiv.style.display = 'flex';
 
+    data.forEach(element => {
+        const itemCard = document.createElement('div');
+        itemCard.classList.add('movie');
+        itemCard.innerHTML = `
+            <span>${element.title}</span>
+            <span>${element.overview}</span>
+        `;
+        resultsDiv.appendChild(itemCard);
+    })
  };
 
 const updatedBackground = function() {
+
     fetch(latestRelease)
     .then(response => response.json())
     .then((data) => {
         let randomNumber = getRandomInt(data.results.length);
         let result = data.results[randomNumber].poster_path;
-        let ultimatePath = path+result;
+        let imageUrl = path+result;
```

Comment:
```suggestion
        const imageUrl = path+result;
```
Esta variable no la estás re-asignando. Igual sucede con las variables que están dentro de esta función.